src/platform/silabs/efr32/wifi/ethernetif.cpp

```diff
@@ -79,6 +94,7 @@ uint32_t gOverrunCount = 0;
  ******************************************************************************/
 static void low_level_init(struct netif * netif)
 {
+    SILABS_LOG("%s: started", __func__);
```

Comment:
remove __func__  and use function name directly

Reply:
Done